src/assets/images/index.js

```diff
@@ -12,6 +12,9 @@ import RedPiece from "./red-piece.png";
 import Snake from "./snake.png";
 import Star from "./star.png";
 
+//**************************** QUESTION *************************
+//Is it considered an object?
+
 const Images = {
   BlueBullet,
```

Comment:
Yes, this is just an object/dictionary that contains the path of the images.

Reply:
I used to see objects in the key-value format. That's why it looked a bit different to me.